samples/dnn/colorization.py

```diff
@@ -27,8 +29,8 @@ def parse_args():
 
     # populate cluster centers as 1x1 convolution kernel
     pts_in_hull = pts_in_hull.transpose().reshape(2, 313, 1, 1)
-    net.getLayer(long(net.getLayerId('class8_ab'))).blobs = [pts_in_hull.astype(np.float32)]
```

Comment:
@catree, It'll break the script at https://github.com/opencv/opencv/blob/master/modules/dnn/misc/python/pyopencv_dnn.hpp#L9. Could you check it and  modify the function to be more flexible?

Reply:
It works now for both Python version.

Reply:
@catree, So, we can emit the casting here?